src/main/java/org/hyperledger/fabric/sdk/Endpoint.java

```diff
@@ -385,90 +475,122 @@ private void addNettyBuilderProps(NettyChannelBuilder channelBuilder, Properties
                 }
             }
 
-            Utils.invokeMethod(channelBuilder, methodName, classParms, parmsArray);
+            try {
+                Utils.invokeMethod(channelBuilder, methodName, classParms, parmsArray);
+            } catch (IllegalAccessException | NoSuchMethodException | InvocationTargetException e) {
+                logger.error(format("Endpoint %s, exception '%s'", url, e.getMessage()), e);
+                logger.error(e);
+                throw new RuntimeException(e);
+            }
```

Comment:
I think this now duplicates the logging done in the Endpoint constructor. It might be simpler to declare these as thrown exceptions and have them handled by the constructor, similar to the original code